UikitDz/UikitDz/SettingsPizzaVC.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  SettingsPizza.swift
+//  UikitDz
+//
+//  Created by Николай Панов on 30.06.2021.
+//
+
+import UIKit
+
+final class SettingsPizzaVC: UIViewController {
+    
+    // MARK: - Public Properties
+    var name = " "
+    var imageName = String()
+    
+    // MARK: - Private Properties
+    private var imagePizza = UIImageView()
+    private let buttonGoToPay = UIButton()
+    private let cheeseLabel = UILabel()
+    private let hamLabel = UILabel()
+    private let mushroomsLabel = UILabel()
+    private let olivesLabel = UILabel()
+    private var namePizza = UILabel()
+    private var cheeseSwitch = UISwitch()
+    private var hamSwitch = UISwitch()
+    private var mushroomsSwitch = UISwitch()
+    private var olivesSwitch = UISwitch()
+    private var cheeseBool = false
```

Comment:
```suggestion
    private var isCheeseBool = false
```


Все булевые переменные называть с is